package.json

```diff
@@ -40,7 +40,6 @@
   },
   "homepage": "https://github.com/Staffbase/plugins-client-sdk#readme",
   "release": {
-    "branch": "master",
```

Comment:
Is this right. I just ask, if you did it for purpose.

Reply:
Yes in the new version, the standard is good enough. It defaults to `['+([0-9])?(.{+([0-9]),x}).x', 'master', 'next', 'next-major', {name: 'beta', prerelease: true}, {name: 'alpha', prerelease: true}]` which gives more possibilities.